lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -238,145 +274,248 @@ bool Caffe2ModelLoader::hasMultidirectionalBroadcast(
   return false;
 }
 
-llvm::Error Caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
-  ArgumentDictionaryTy dict = loadArgumentMap(op);
+llvm::Error Caffe2ModelLoader::loadConv(const caffe2::OperatorDef &op,
+                                        ArgumentDictionaryTy &dict) {
+  const std::string &opName = loadOperatorName(op);
   const std::string &typeName = op.type();
```

Comment:
nit: Given `typeName` is just used once now and much lower in this function, I'd move this down to just before its use, or just use `op.type()` directly at its use site to make clear it has no impact on any logic up here.